deploy/charts/harvester/values.yaml

```diff
@@ -413,6 +413,11 @@ webhook:
 
   httpsPort: 9443
 
+upgrade:
+  image:
+    repository: rancher/harvester-upgrade
+    tag: master-head
```

Comment:
I think the ISO build would fail if the image does not exist, seem we can only merge this PR after the repo is prepared.